package.json

```diff
@@ -85,14 +87,15 @@
     "@bigtest/mirage": "^0.0.1",
     "@bigtest/mocha": "^0.5.2",
     "@bigtest/react": "^0.1.2",
-    "@folio/eslint-config-stripes": "^3.2.1",
-    "@folio/stripes": "^2.3.0",
+    "@folio/eslint-config-stripes": "^5.1.0",
+    "@folio/stripes-components": "~5.10.0",
+    "@folio/stripes": "^2.13.0",
     "@folio/stripes-cli": "^1.11.0",
-    "@folio/stripes-core": "^3.3.0",
-    "babel-eslint": "^9.0.0",
+    "@folio/stripes-core": "^3.12.0",
+    "babel-eslint": "^10.0.3",
     "babel-polyfill": "^6.26.0",
```

Comment:
Could you please get rid of polyfill in favor of 

```
"regenerator-runtime": "^0.13.3",
"core-js": "^3.6.4",
```

See the usage [here](https://github.com/folio-org/ui-checkin/pull/267/files#diff-cb86011ded27b35eec449893a2b6a47cR1).